### ISSUE TITLE
True comments in Maptool?

### ISSUE THREAD

**Issue by Merudo** [RPTools/maptool#730]
**Is your feature request related to a problem? Please describe.**
It's kind of difficult and tricky to create comments in maptool macros.

Using `<--` and `-->` by themselves will sometimes create issues, especially if there are macro commands commented out.

**Describe the solution you'd like**
A true way to have comments in Maptool. Ideally, the element "//" would comment out the rest of a line.

**Comment by Phergus** [comment]
Past discussions of this suggest this can't happen with the current parser.

**Comment by aliasmask** [comment]
In coldfusion, parser comments are  (an extra dash on both ends).

**Comment by Phergus** [comment]
Pretty sure that the new parser will support true comments.  @kayila ?

**Comment by kayila** [comment]
Yes. The new parser supports both // and /* ... */ Style comments

**Comment by JamzTheMan** [comment]
And assume those are truly stripped out and ignored (not parsed) unlike HTML comments are today? :)

Can't wait until I can just block comment out a suspect section of code...

**Comment by kayila** [comment]
Correct, the parser actually throws the comment out entirely when parsing
